spec-main/api-browser-window-spec.ts

```diff
@@ -1257,6 +1256,14 @@ describe('BrowserWindow module', () => {
         expect(w.getOpacity()).to.equal(1.0)
       }).to.not.throw()
     })
+
+    it('does not change opacity if given number out of bounds', () => {
```

Comment:
Well, we _do_ change opacity if the number is out of bounds, we just bring it in bounds first

```suggestion
    it('clamps opacity to [0.0...1.0]', () => {
```